aten/src/THC/THCCachingAllocator.cpp

```diff
@@ -161,6 +167,9 @@ struct THCCachingAllocator
     allocated_blocks[block->ptr] = block;
 
     *devPtr = (void*)block->ptr;
+
+    memory_allocated += block->size;
+    max_memory_allocated = std::max(max_memory_allocated, memory_allocated);
```

Comment:
Please include `<algorithm>` before using this in MSVC.

Reply:
will do. thanks!